README.md

```diff
@@ -106,17 +106,24 @@ You can use this simple function to add and remove the animations:
 
 ```javascript
 function animateCSS(element, animationName, callback) {
-    const node = document.querySelector(element)
-    node.classList.add('animated', animationName)
+    const node = document.querySelector(element);
+    let classes = animationName.split(" ");
+
+    classes.forEach((v,i)=>{
```

Comment:
How about change the var `v` to a more descriptive name.
No need for the index here.